bundles/plugins/org.bonitasoft.studio.common/src/org/bonitasoft/studio/common/properties/PropertySectionWithTabs.java

```diff
@@ -1,19 +1,16 @@
 /**
  * Copyright (C) 2010 BonitaSoft S.A.
```

Comment:
Do we want to update existing copyrights ?
If yes we should do it once and for all.